packages/react/src/components/Button/Button.Skeleton.js

```diff
@@ -7,20 +7,22 @@
 
 import PropTypes from 'prop-types';
 import React from 'react';
-import classNames from 'classnames';
+import cx from 'classnames';
 import { settings } from 'carbon-components';
 
 const { prefix } = settings;
 
-const ButtonSkeleton = ({ small, href }) => {
-  const buttonClasses = classNames({
+const ButtonSkeleton = ({ className, small, href, ...rest }) => {
+  const buttonClasses = cx({
     [`${prefix}--skeleton`]: true,
     [`${prefix}--btn`]: true,
     [`${prefix}--btn--sm`]: small,
+    className,
```

Comment:
```suggestion
    [className]: className,
```

Reply:
Oh oops, so this one should be outside of the brackets actually. 😅 Shouldn't need a key then?

Reply:
both would result in the same thing, it's just that if you want to place it in an object you'll need to compute the key name

Reply:
🤔 I'm not sure what you mean. My change passes it as an argument, outside of the object, as is done elsewhere in this PR and in the codebase. (i.e., https://www.npmjs.com/package/classnames#usage)